pkg/vault/test/foundry/utils/BaseERC4626BufferTest.sol

```diff
@@ -30,32 +30,106 @@ abstract contract BaseERC4626BufferTest is BaseVaultTest {
     uint256 internal waUsdcIdx;
     address erc4626Pool;
 
+    // Rounding issues are introduced when dealing with tokens with rates different than 1:1. For example, to scale the
+    // tokens of an yield-bearing pool, the amount of tokens is multiplied by the rate of the token, which is
+    // calculated using `convertToAssets(FixedPoint.ONE)`. It generates an 18 decimal rate, but quantities bigger than
+    // 1e18 will have rounding issues. Another example is the different between convert (used to calculate query
+    // results of buffer operations) and the actual operation.
+    uint256 internal errorTolerance = 1e8;
```

Comment:
Is this 1e-10%? (0.000000001%?) We haven't usually used assertApproxEqRel (usually assertApproxEqAbs), so consider renaming to show that this is a percentage (same as ours, where 1e18 = 100%, according to the docs).

```suggestion
    uint256 internal errorTolerancePercentage = 1e8; // 1e18 = 100%
```

Reply:
Since the BPT amount is also 18 decimals, the difference between Absolute and Relative is small. So, I just changed the test to Abs.